Explorer/Assets/DCL/SDKComponents/MediaStream/Systems/UpdateMediaPlayerSystem.cs

```diff
@@ -73,12 +74,20 @@ private void OnMasterVolumeChanged(float volume)
 
         protected override void Update(float t)
         {
+            UpdateMediaPlayerPositionQuery(World);
             UpdateAudioStreamQuery(World);
             UpdateVideoStreamQuery(World);
 
             UpdateVideoTextureQuery(World);
         }
 
+        [Query]
+        private void UpdateMediaPlayerPosition(ref MediaPlayerComponent mediaPlayer, ref TransformComponent transformComponent)
+        {
+            // Needed for positional sound
+            mediaPlayer.MediaPlayer.transform.position = transformComponent.Transform.position;
```

Comment:
Can you use cached position instead to avoid multiple access to `transform`?

Reply:
Is there a cached position somewhere? If you mean that I should cache it, I don't see how, I need the current position of every entity.

Reply:
in `TransformComponent` itself